rb/spec/integration/selenium/webdriver/timeout_spec.rb

```diff
@@ -34,7 +34,7 @@ module WebDriver
           driver.manage.timeouts.implicit_wait = 6
 
           driver.find_element(id: 'adder').click
-          driver.find_element(id: 'box0')
+          expect { driver.find_element(id: 'box0') }.not_to raise_error(WebDriver::Error::NoSuchElementError)
```

Comment:
This should just be `.not_to raise_error` otherwise it potentially hides errors

Reply:
Thank you for your advice. Fixed it.